server/svix-server/src/v1/endpoints/message.rs

```diff
@@ -263,6 +265,14 @@ async fn create_message(
     // Should never happen since you're giving it an existing Application, but just in case
     .ok_or_else(|| err_generic!("Application doesn't exist: {}", app.id))?;
 
+    if insert_event_type.unwrap_or(false) {
+        if let serde_json::Value::Object(ref mut obj) = data.payload {
+            let value = serde_json::to_value(data.event_type.clone())
```

Comment:
This looks expensive, what doss it do?

Reply:
Ahh, I'm silly, I can just construct the `serde_json::Value::String` directly, good catch.